graphql_query.py

```diff
@@ -37,35 +37,39 @@ def auth():
 
 
 def query_event(event_id: str):
-    query = (
-        "query($eventId: ID) {\n"
-        "event(id: $eventId) {\n"
-        "id\n"
-        "title\n"
-        "eventUrl\n"
-        "description\n"
-        "shortDescription\n"
-        "group {\n"
-        "id\n"
-        "name }\n"
-        "isOnline\n"
-        "eventType\n"
-        "venue {\n"
-        "id\n"
-        "name\n"
-        "address\n"
-        "city\n"
-        "postalCode\n"
-        "lng\n"
-        "lat }\n"
-        "onlineVenue {\n"
-        "url }\n"
-        "dateTime\n"
-        "duration\n"
-        "timezone\n"
-        "endTime\n"
-        "}\n}"
-    )
+    query = """
+    query ($eventId: ID) {
+      event(id: $eventId) {
+        id
+        title
+        eventUrl
+        description
+        shortDescription
+        group {
+          id
+          name
+        }
+        isOnline
+        eventType
+        venue {
+          id
+          name
+          address
+          city
+          postalCode
+          lng
+          lat
+        }
+        onlineVenue {
+          url
+        }
+        dateTime
+        duration
+        timezone
+        endTime
+      }
+    }
+    """
     token = auth().get("access_token")
```

Comment:
Should we make it mandatory, to avoid risking passing `token=None`?

```suggestion
    token = auth()["access_token"]
```